### PR TITLE
Add the ability to include a single file

### PR DESCRIPTION
**Pull request by ahmyi**
```extra.preload.files```

Add the ability from configuration to allow a single file or multiple files to be included.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,7 +161,7 @@ yourself. Then, call all of the generated `vendor/preload.php` files.
 
 By default, the preload file will contain a small snippet at the top that will quit the script immediately if Opcache
 is not available. If you plan to include this `vendor/preload.php` file from another script, you can use the special
-command line option `composer preload --no-status-check` that will make the `vendor/preload.php` file not contain these
+command line option `composer preoad --no-status-check` that will make the `vendor/preload.php` file not contain these
```
Comment (Ayesh) [inline README.md]: We got a typo here with "preoad" missing "l".

File: README.md [hunk 2]
```diff
@@ -103,6 +106,11 @@ If this setting is set to `true` (you can also pass command line option `--no-st
 default, and the generated `preload.php` file will contain a small snippet on the top that makes it quit if opcache is 
 not enabled.
 
+ - `extra.preload.paths` : _Optional_
+
+An array of single `.php` files to be included. This setting is optional. The files must exist
```
Comment (Ayesh) [inline README.md]: I don't think we have to say `.php` files. Drupal, for example, has PHP files with extensions such as `.module` and `.install`. I think we can just say `An array of individual PHP files to...`. Do you agree?

File: README.md [hunk 3]
```diff
@@ -106,7 +106,7 @@ If this setting is set to `true` (you can also pass command line option `--no-st
 default, and the generated `preload.php` file will contain a small snippet on the top that makes it quit if opcache is 
 not enabled.
 
- - `extra.preload.paths` : _Optional_
+ - `extra.preload.files` : _Optional_
```
Comment (Ayesh) [inline README.md:109]: We shouldn't remove the `paths` configuration as it is already implemented. Can we _add_ `files` instead of replacing?

**Review by Ayesh — CHANGES_REQUESTED** [review]
Hello @ahmyi, thank you for the PR. Including individual files in the preload is not a use case I had thought about, and now with your PR, I can certainly see the importance of it! Thank you. 
I have made a few minor comments in the PR, which would help if you could amend to the same PR. I am currently away from my computer and these reviews were made on my phone. I will merge the PR as soon as I get to my laptop, and make the changes myself if they are not in the PR already. 

Thank you.

File: README.md (hunk 2 above)
Comment (ahmyi) [reply README.md]: True i agree

File: README.md (hunk 3 above)
Comment (ahmyi) [reply README.md:109]: I don't understand. I haven't removed paths configuration i add in files

Comment (ahmyi) [reply README.md:109]: Oh. I had a typo from my previous commit which was a duplication of ```extra.preload.paths``` config. It was supposed to be files.

**Review by Ayesh — APPROVED** [review]
Thank you @ahmyi . I will be cutting a new release next week or so, and this change will be included in it. 
Any further improvements you have in mind will be welcome as well.

**Pull request merged**